### PR TITLE
feat(llgoexamples/cpp/spdlog):Added a C++ logging library

### PR DESCRIPTION
**Pull request by 2458zhanghehe**

### CODE REVIEW

File: cpp/spdlog/spdlog.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+	"github.com/goplus/llgo/c"
+)
+
+const (
+	LLGoPackage = "link: $(pkg-config --libs cppWrap spdlog); -lcppWrap -lspdlog -pthread -lfmt"
+)
+
+//go:linkname PrintInfo C.PrintInfo
+func PrintInfo(msg *c.Char)
```
Comment (xushiwei) [inline cpp/spdlog/spdlog.go:15]: should have namespace: eg. `spdlogPrintInfo`

File: cpp/spdlog/spdlog.go [hunk 2]
```diff
@@ -0,0 +1,35 @@
+package spdlog
+
+import (
+	_ "unsafe"
+
+	"github.com/goplus/llgo/c"
+)
+
+const (
+	LLGoPackage = "link: $(pkg-config --libs cppWrap spdlog); -lcppWrap -lspdlog -pthread -lfmt"
```
Comment (xushiwei) [inline cpp/spdlog/spdlog.go]: cppWrap don't need to build as a library. You can use `LLGoFiles`.

File: cpp/spdlog/spdlog.go (hunk 1 above)
Comment (2458zhanghehe) [reply cpp/spdlog/spdlog.go:15]: ok, the func name in code has change to spdlogFunc

File: cpp/spdlog/spdlog.go (hunk 2 above)
Comment (2458zhanghehe) [reply cpp/spdlog/spdlog.go]: ok, the code has changed from generating dylib to use LLGoFiles

File: cpp/spdlog/spdlog.go (hunk 1 above)
Comment (2458zhanghehe) [reply cpp/spdlog/spdlog.go:15]: when added nameSpace, the qiniu-x remind:
error

have removed namespace

**Pull request closed without merging**